packages/synapse-react-client/src/components/QueryWrapper/QueryWrapper.test.tsx

```diff
@@ -5,53 +5,52 @@ import {
   LockedColumn,
   QueryContextType,
   useQueryContext,
-} from '../../src/components/QueryContext/QueryContext'
-import {
-  QueryWrapper,
-  QueryWrapperProps,
-} from '../../src/components/QueryWrapper/QueryWrapper'
-import { SynapseConstants } from '../../src/utils'
+} from '../QueryContext'
+import { QueryWrapper, QueryWrapperProps } from './QueryWrapper'
+import { SynapseConstants } from '../../utils'
 import {
   QueryBundleRequest,
   QueryResultBundle,
+  Row,
 } from '@sage-bionetworks/synapse-types'
-import syn16787123Json from '../../src/mocks/query/syn16787123'
-import { DEFAULT_PAGE_SIZE } from '../../src/utils/SynapseConstants'
-import useTableRowSelection from '../../src/components/QueryWrapper/useTableRowSelection'
-import SynapseClient from '../../src/synapse-client'
-import { mockCompleteAsyncJob } from '../../src/mocks/mockFileViewQuery'
+import syn16787123Json from '../../mocks/query/syn16787123'
+import { DEFAULT_PAGE_SIZE } from '../../utils/SynapseConstants'
+import SynapseClient from '../../synapse-client'
+import { mockCompleteAsyncJob } from '../../mocks/mockFileViewQuery'
 import userEvent from '@testing-library/user-event'
-import { createWrapper } from '../../src/testutils/TestingLibraryUtils'
+import { createWrapper } from '../../testutils/TestingLibraryUtils'
 
-jest.mock('../../src/synapse-client', () => ({
+jest.mock('../../synapse-client', () => ({
   getQueryTableAsyncJobResults: jest.fn(),
   getEntity: jest.fn(),
 }))
 
-jest.mock('../../src/components/QueryWrapper/useTableRowSelection', () => ({
-  __esModule: true,
-  default: jest.fn(),
-}))
-
 const mockGetQueryTableAsyncJobResults = jest.mocked(
   SynapseClient.getQueryTableAsyncJobResults,
 )
-const mockUseTableRowSelection = jest.mocked(useTableRowSelection)
 
 let providedContext: QueryContextType | undefined
 const renderedTextConfirmation = 'QueryWrapper rendered!'
+let isLoadingNewBundleValue: boolean | undefined
+let currentQueryDataValue: QueryResultBundle | undefined
+let selectedRows: Row[] | undefined
+let setSelectedRows: QueryContextType['setSelectedRows'] | undefined
 
-const QueryContextReceiver = jest.fn((props: any) => {
+const QueryContextReceiver = () => {
   // An error would be thrown if context was not provided by QueryWrapper
   const context = useQueryContext()
+  isLoadingNewBundleValue = context.isLoadingNewBundle
+  currentQueryDataValue = context.data
+  selectedRows = context.selectedRows
+  setSelectedRows = context.setSelectedRows
```

Comment:
Having individual variables for different state items will make more sense when the new state manager is pulled in. This is sort of "intermediary" code that works with the existing context but can be easily changed